mlflow/fastai/__init__.py

```diff
@@ -190,6 +193,10 @@ def save_model(
         data=model_data_subpath,
         code=code_dir_subpath,
     )
+    try:
+        mlflow_model.model_size_bytes = get_total_file_size(path)
+    except Exception as e:
+        _logger.info(f"Failed to get the total size of {path} because of error :{e}")
```

Comment:
Can we warn in ``get_total_file_size` and returns None if it fails?

Reply:
I thought about try-catching in `get_total_file_size` v.s. doing it here, and slightly prefer doing it here because
* The logic is more concise, as otherwise we'll need both a try-catching and a `if size is not None` block
* Using the logger for each module gives more information

Let me know if you have other opinions

Reply:
I thought about something like this:

```suggestion
    if size := get_total_file_size(path):
        mlflow_model.model_size_bytes = size
        
        
def get_total_file_size(...):
    try:
        ...
        return size
    except:
        _logger.warning("Failed")
        return None
```

Reply:
ahh right - forgot there's the new `:=` syntax.
So I don't really have a strong opinion between the two implementations... I can change if you prefer this more (or if you're ok with both, I'll just be lazy to keep as it is)

Reply:
I'd prefer my approach because it looks cleaner and the warning doesn't seem to have to be fired from the flavor module.